utils/GenUtils.js

```diff
@@ -60,10 +61,10 @@ function compareJsonStrings(str1, str2) {
  * @param {number} mode  0 - bits mode,  1 - bytes mode
  * @returns - string representation of data speed
  */
-function bytesSpeedToString(amount, mode = 1) {
+function bytesSpeedToString(amount, mode = appSettingsModel.displayBytes) {
```

Comment:
Can you pass some hard coded value true.
This is reusable code, Idea is not to add model dependency on Utils.